doc/Hammer-Basics/Hammer-Setup.rst

```diff
@@ -5,11 +5,7 @@ Hammer has a few requirements and there are several environment variables to set
 
 System Requirements
 -----------------------------
-- Python 3.6+ recommended (minimum Python 3.3+)
-
--- For Python 3.4 and lower, the ``typing`` module must be installed. (``python3 -m pip install typing``)
-
--- For Python 3.4, the enum34 package must be installed. (``python3 -m pip install enum34``)
+- Python 3.6+ recommended (minimum Python 3.5+)
```

Comment:
Ah, oops I made a typo in my comment, f-strings are only supported for Python 3.6+. #656 actually updates this file so you might as well remove this change and wait to merge from master after that PR goes through.